context.go

```diff
@@ -10,18 +10,21 @@ import (
 	"github.com/goravel/framework/contracts/http"
 )
 
+const goravelContextKey = "goravel_contextKey"
+
 func Background() http.Context {
 	ctx, _ := gin.CreateTestContext(httptest.NewRecorder())
 	return NewContext(ctx)
 }
 
 type Context struct {
+	ctx      context.Context
```

Comment:
It can be removed.

Reply:
done in https://github.com/goravel/gin/pull/88/commits/5eeae94d7c4edb4dccd44133ca45f00fab416196